frontstage_api/resources/passwords/verify_password_token.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from flask import request
+from flask_restplus import Resource
+from structlog import wrap_logger
+
+from frontstage_api import api, auth
+from frontstage_api.controllers import party_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@api.route('/verify_password_token')
+class VerifyPasswordToken(Resource):
+
+    @staticmethod
+    @auth.login_required
+    def get():
+        logger.info('Attempting verify password token')
+        token = request.args.get('token')
+
+        party_controller.verify_token(token)
+        logger.info('Successfully verified password token')
+
+        return "OK", 200
```

Comment:
Do need to return the text "OK"? would the status code be good enough?

Reply:
This is copied from some of the code I did, this was my method of making an 'empty' response with the 200 status code. If there's an easy way to just send 200 that is better for sure

Reply:
Maybe return Response(status_code=200)?